### PR TITLE
Text To LANS in FAQs #6 

### PR DESCRIPTION
**Pull request by MikeyGMT**
Text To LANS in FAQs #6 
Also split prefs into 3 tabs ... General, FAQs and Categories so there are less Lan Defines needed.
There is more LANS work TODO, but let's do that in another pull.

### CODE REVIEW

**Comment by MikeyGMT** [comment]
Testing

image

**Comment by Moc** [comment]
LAN_REMOVE may be added as a common term I think, or we can replace it with LAN_DELETE? Depending on the context. 

Great work with the LAN's Mike!

File: e107_plugins/faqs/faqs.php [hunk 1]
```diff
@@ -333,7 +329,7 @@ function view_all_query($srch='')
 			$insert = " AND (f.faq_question LIKE '%".$srch."%' OR f.faq_answer LIKE '%".$srch."%' OR FIND_IN_SET ('".$srch."', f.faq_tags) ) ";
 
 
-			$message = "<span class='label label-lg label-info'>".$srch." <a class='e-tip' title='Remove' href='".$removeUrl."'>×</a></span>";
```
Comment (Moc) [inline e107_plugins/faqs/faqs.php]: LAN_REMOVE may be added as a common term I think, or we can replace it with LAN_DELETE? Depending on the context.

Comment (MikeyGMT) [inline e107_plugins/faqs/faqs.php]: Not sure where to access - cannot see a "remove" button in the FAQ anywhere - suspect may be legacy code ... can you give a clue?

**Pull request merged**